app/services/to_cocina/work/description_mapper.rb

```diff
@@ -45,14 +45,21 @@ def params
       end
       # rubocop:enable Metrics/AbcSize
 
+      # rubocop:disable Metrics/AbcSize
```

Comment:
If you move this after `def note_params` on the next line, you can remove the explicit re-enable on line 62.